ui/src/domain/identity.ts

```diff
@@ -3,33 +3,32 @@ export enum IdentityType {
   ETH = "ETH",
 }
 
-export enum CredentialStatusType {
-  "Iden3OnchainSparseMerkleTreeProof2023" = "Iden3OnchainSparseMerkleTreeProof2023",
-  "Iden3ReverseSparseMerkleTreeProof" = "Iden3ReverseSparseMerkleTreeProof",
-  "Iden3commRevocationStatusV1.0" = "Iden3commRevocationStatusV1.0",
-}
-
 export enum Method {
   iden3 = "iden3",
   polygonid = "polygonid",
 }
 
+export type Network = {
+  name: string;
+  rhsMode: [string, ...string[]];
+};
+
 export type SupportedNetwork = {
   blockchain: string;
-  networks: [string, ...string[]];
+  networks: [Network, ...Network[]];
```

Comment:
Having the type `SupportedNetwork` that has the property `networks` is very confusing.

Let's rename the `SupportedNetwork` type as `Blockchain`, so we have a blockchain entity (Polygon) that has networks on it (Mainet, Amoy, etc).

Please make sure you propagate these renames to all the parsers, constants, function parameters, etc.